### PR TITLE
Event refactoring

### LINKED ISSUE #54: Event teaser - Revamp and finish
flickity

- [x] basic

- [x] highlight

- [x] just finished 

- [x] invitation

### PR DESCRIPTION
**Pull request by Yago**
will close  #54

### CODE REVIEW

File: assets/components/molecules/card-slider/card-slider.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      const flkty = new Flickity('.card-slider', {
+        cellAlign: 'left',
+        setGallerySize: true,
+        pageDots: isMobile,
+        prevNextButtons: false,
+        contain: true,
+        groupCells: false,
+      });
+
+      $('.card-slider-cell').css('height', '100%');
```
Comment (tonifisler) [inline assets/components/molecules/card-slider/card-slider.js]: Cannot be achieved with CSS?

File: assets/components/molecules/card/card.scss [hunk 2]
```diff
@@ -8,6 +8,14 @@
   margin-bottom: $grid-gutter-width;
 }
 
+a.card.bg-gray-100 {
```
Comment (tonifisler) [inline assets/components/molecules/card/card.scss]: This is too specific and might not have the result we want with cards outside of this component. I guess a class would be better in that case.

File: assets/components/organisms/event/event-highlight.twig [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{% set img = img|default(true) %}
```
Comment (tonifisler) [inline assets/components/organisms/event/event-highlight.twig]: What is this for?

**Comment by tonifisler** [comment]
- I fixed some issues with the breakpoints (between SM and MD, no flickity-dots or pren+next buttons were displayed) 
- I set a min-width for the cards in the slider, it was too small and words were breaking:
  
screen shot 2018-06-12 at 10 02 22

File: assets/components/molecules/card-slider/card-slider.js (hunk 1 above)
Comment (Yago) [reply assets/components/molecules/card-slider/card-slider.js]: No and it's a JS widget who doesn't without it anyway.

**Comment by Yago** [comment]
@tonifisler it's ready !

File: assets/components/organisms/event/event-just-finished.twig [hunk 4]
```diff
@@ -1 +1,8 @@
-<img src="./images/wip-screenshot/event-just-finished.png" class="img-fluid" />
+{% extends "@organisms/event/event.twig" %}
+
+{% block classes %}card card-grayscale link-trapeze-horizontal bg-gray-100{% endblock %}
```
Comment (tonifisler) [inline assets/components/organisms/event/event-just-finished.twig]: Should it be also switched to `.card-gray`? Impacts the background-color on hover.

Comment (Yago) [reply assets/components/organisms/event/event-just-finished.twig]: Not swtiched, but work on a better correlation 😉

**Pull request merged**